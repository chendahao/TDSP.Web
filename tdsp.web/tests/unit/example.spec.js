import { shallowMount } from '@vue/test-utils'
import home from '@/components/Homeindex'

describe('home.vue', () => {
  it('renders props.msg when passed', () => {
    const wrapper = shallowMount(home, {
      stubs: ['router-link', 'v-container', 'v-layout', 'v-flex', 'v-img']
    })
    expect(wrapper.html()).toMatchSnapshot()
  })
})
