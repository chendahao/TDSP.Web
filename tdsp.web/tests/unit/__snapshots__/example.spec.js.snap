// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`home.vue renders props.msg when passed 1`] = `
<v-container-stub>
  <v-layout-stub text-xs-center="" wrap="">
    <v-flex-stub xs12="">
      <v-img-stub src="" contain="" height="200" class="my-3"></v-img-stub>
    </v-flex-stub>
    <v-flex-stub mb-4="">
      <h1 class="display-2 font-weight-bold mb-3">
        欢迎使用VTS MIS系统
      </h1>
    </v-flex-stub>
  </v-layout-stub>
</v-container-stub>
`;
